### ISSUE TITLE
Transparency slider

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#140]
```
Currently MinTTY has 4 radio buttons for degrees of transparency: Off, Low,
Medium, and High.

It would be nice if instead it had a slider and textbox, allowing custom
degrees of transparency. It would also be useful if you could see the
window change as you adjusted the transparency settings so that you could
fine-tune your settings without having to exit and re-invoke a modal dialog.
```

Original issue reported on code.google.com by `stem...@gmail.com` on 21 Oct 2009 at 9:40

**Comment by mintty** [comment]
Released 2.7.5 with hotkeys (Ctrl+Shift+T+cursor-keys) to interactively adjust transparency.

**Comment by mintty** [comment]
Released 3.5.1.
